packages/evm/src/interpreter.ts

```diff
@@ -10,6 +10,7 @@ import { trap } from './opcodes'
 import { Stack } from './stack'
 
 import type { EVM, EVMResult } from './evm'
+import type { Journal } from './journal'
 import type { AsyncOpHandler, OpHandler, Opcode } from './opcodes'
 import type { Block, Blockchain, Log } from './types'
 import type { Common, EVMStateManagerInterface } from '@ethereumjs/common'
```

Comment:
Debug logger from below, currently `evm:eei:gas` (so annying this untouched-lines-comment-deactivation), should be renamed.

Simply `evm:gas`?

Reply:
Yes, `evm:gas` is perfect. I will do a quick check to see if there are any other eei leftovers.